addons/sale_timesheet/models/account.py

```diff
@@ -71,9 +71,9 @@ def _check_can_write(self, values):
     def _timesheet_preprocess(self, values):
         values = super(AccountAnalyticLine, self)._timesheet_preprocess(values)
         # task implies so line (at create)
-        if 'task_id' in values and not values.get('so_line') and values.get('employee_id'):
+        if 'task_id' in values and not values.get('so_line') and (values.get('employee_id') or self.employee_id):
```

Comment:
I guess `hr_timesheet` will always be installed? Otherwise `employee_id` won't be on `self`

Reply:
Indeed it the module sale_timesheet depends on hr_timesheet